pkgs/development/python-modules/pypi-sources.json

```diff
@@ -0,0 +1,80 @@
+{
+  "toolz": {
+    "latest_version": "0.7.4",
+    "meta": {
```

Comment:
in the past i found that metadata (license in my case) changed from version to version. it would be better to have all metadata copied per version.

i'm not 100% for this, but if i find the package with this weird thing i'll link it here...